ta/pkcs11/src/pkcs11_helpers.h

```diff
@@ -55,6 +61,13 @@ bool key_type_is_symm_key(uint32_t key_type_id);
 /* Return true if the key type @key_type_id relates to an asymmetric key */
 bool key_type_is_asymm_key(uint32_t key_type_id);
 
+/* Calculate AES key check value size */
+enum pkcs11_rc aes_get_check_value_size(void *key, uint32_t key_size,
+					uint32_t *kcv_size);
```

Comment:
I think it's easier to remove this function and move `PKCS11_CKA_CHECK_VALUE_SIZE` definition to _pkcs11_attributes.h_, the size applies to all objects (certificate and symm keys).